mico-core/src/main/java/io/github/ust/mico/core/web/ApplicationController.java

```diff
@@ -134,14 +155,14 @@
         }
 
         // Including services must not be updated through this API. There is an own API for that purpose.
-        if (application.getServices().size() > 0) {
+        List<MicoService> services = serviceRepository.findAllByApplication(application.getShortName(), application.getVersion());
```

Comment:
Same here: We want to get the services that are included in the request body.

Reply:
Done.